libs/sdk/examples/exsat_network/spin.toml

```diff
@@ -0,0 +1,27 @@
+spin_manifest_version = 2
+
+[application]
+authors = ["Yordan Madzhunkov"]
+name = "Exsat network btc holdings"
+version = "0.1.0"
+
+[application.trigger.settings]
+interval_time_in_seconds = 30
+sequencer = "http://127.0.0.1:9856/post_reports_batch"
+secret_key = "536d1f9d97166eba5ff0efb8cc8dbeb856fb13d2d126ed1efc761e9955014003"
+reporter_id = 0
+
+[[trigger.oracle]]
+component = "exsat-holdings"
+
+[[trigger.oracle.data_feeds]]
+id = "606"
```

Comment:
This ID is just for testing purposes, we need to change it to match the deployment